tests/query/processors/test_mandatory_condition_applier.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+
+# from snuba.clickhouse.columns import ColumnSet, String
+
+from snuba.datasets.schemas.tables import TableSource
+from snuba.query.logical import Query
+from snuba.query.processors.mandatory_condition_applier import MandatoryConditionApplier
+
+from snuba.request.request_settings import HTTPRequestSettings
+
+
+test_data = [
+    ("table1", [["time1", "=", "1"]]),
+    ("table2", [["time2", "=", "2"]]),
+]
+
+
+@pytest.mark.parametrize("table, mand_condition", test_data)
+def test_prewhere(table, mand_condition) -> None:
```

Comment:
nit: this is not the prewhere test anymore :D. Could you please rename it ?
(and also add type hints for your parameters)